### PR TITLE
chore(execution-engine): a set of trace-unrelated negative tests for UncatableError [fixes VM-263]

### PR DESCRIPTION
**Pull request by raftedproc**
chore(execution-engine): a set of trace-unrelated negative tests for UncatableError [fixes VM-263]

### CODE REVIEW

File: air/tests/test_module/negative_tests/uncatchable_trace_unrelated.rs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    )
+    "#);
+
+    let result = peer_vm_1.call(script, "", "", <_>::default()).unwrap();
+    let expected_error = ShadowingIsNotAllowed(var_name);
+    assert!(check_error(&result, expected_error));
+}
+
+#[test]
+fn value_for_cid_not_found() {
```
Comment (monoid) [inline air/tests/test_module/negative_tests/uncatchable_trace_unrelated.rs:103]: Actually, tests for this error exists at `air/tests/test_module/features/cid/mod.rs`.

**Review by mikevoronov — APPROVED** [review]
utACK

**Comment by mikevoronov** [comment]
Let's remove `InvalidCanonStreamInData` in a separate PR.

File: air/tests/test_module/negative_tests/uncatchable_trace_unrelated.rs (hunk 1 above)
Comment (raftedproc) [reply air/tests/test_module/negative_tests/uncatchable_trace_unrelated.rs:103]: It does indeed. I just put it here w/o removing it from mod.

**Pull request merged**